src/web_connection_base.cpp

```diff
@@ -113,7 +113,11 @@ namespace libtorrent
 
 	void web_connection_base::start()
 	{
-		set_upload_only(true);
```

Comment:
I seem to recall there's some accounting in the torrent of the number of connected seeds, won't this cause discrepancies there?